.travis.yml

```diff
@@ -174,6 +174,7 @@ env:
              component/test_stopped_vm
              component/test_tags
              component/test_templates
+             component/test_template_access_across_domains
```

Comment:
```suggestion
```

@nvazquez 
let's remove the test from travis and add it back when it is fixed ?

Reply:
+1

Reply:
@soreana can you please add this temporary fix? Then we can add it back when fixed

Reply:
@soreana 
I have committed the change , now we can go forward and merge it if travis tests pass.

cc @nvazquez